assets/js/topics.js

```diff
@@ -48,10 +48,10 @@ $(function () {
     $("#tree").fancytree({
         extensions: ["filter", "glyph"],
         types: {
-            "topic": {icon: "glyphicon glyphicon-folder-close"},
-            "concept": {icon: "glyphicon glyphicon-asterisk"},
-            "variable": {icon: "glyphicon glyphicon-stats"},
-            "question": {icon: "glyphicon glyphicon-question-sign"},
+            "topic": {icon: "fas fa-folder"},
```

Comment:
Please replace the icons with the ones described here:
https://github.com/ddionrails/ddionrails/issues/729#issuecomment-513154371